tests/main/snapd-snap/task.yaml

```diff
@@ -9,18 +9,18 @@ priority: 100
 restore: |
     cd "$PROJECT_PATH"
     echo "Cleanup build artifacts"
-    not command -v snapcraft >/dev/null || snapcraft clean
+    rm -rf parts
```

Comment:
not sure why this change, if there was no snapcraft there would be no parts either, no?

Reply:
This was because of issues with "clean" when `--destructive-mode` flag was used for the build, and before realizing `clean` also accepts `--destructive-mode` flag. Fixed.